packages/@aws-cdk/aws-lambda-nodejs/test/integ-handlers/dependencies-sdk-v3.ts

```diff
@@ -0,0 +1,9 @@
+/* eslint-disable no-console */
+// @ts-ignore
```

Comment:
```suggestion
```

Reply:
I tried to remove this line. But it make build an error as following:
```
test/integ-handlers/dependencies-sdk-v3.ts:2:26 - error TS2307: Cannot find module '@aws-sdk/client-s3' or its corresponding type declarations.

2 import { S3Client } from '@aws-sdk/client-s3'; // eslint-disable-line import/no-extraneous-dependencies, import/no-unresolved
                           ~~~~~~~~~~~~~~~~~~~~
```

So I added this line again, because I think it is better than adding `@aws-sdk/client-s3` to dependencies. WDYT?